AggregateMetrics/AggregateMetrics/AzureWebApp/PerformanceCollectorModule.cs

```diff
@@ -12,6 +12,8 @@
     /// </summary>
     public class PerformanceCollectorModule : ITelemetryModule
     {
+        internal string sdkVersion = SdkVersionUtils.VersionPrefix + SdkVersionUtils.GetAssemblyVersion();
```

Comment:
this should be separate prefix specific for azure web apps performance counters, not one shared with aggregate module. I don't see where you've declared it